Mathlib/Analysis/SpecialFunctions/Exponential.lean

```diff
@@ -421,3 +421,14 @@ theorem hasDerivAt_exp_smul_const' (x : 𝔸) (t : 𝕂) :
 end RCLike
 
 end exp_smul
+
+section tsum_tprod
+
+variable {𝕂 𝔸 : Type*} [RCLike 𝕂] [NormedCommRing 𝔸] [NormedAlgebra 𝕂 𝔸] [CompleteSpace 𝔸]
+
+/-- If `f` has sum `a`, then `exp ∘ f` has product `exp a`. -/
+lemma HasSum.NormedSpace_exp {ι : Type*} {f : ι → 𝔸} {a : 𝔸} (h : HasSum f a) :
```

Comment:
ditto

Reply:
`NormedSpace.hasSum_exp` perhaps? Although this would make dot notation on `HasSum` unavailable here.

Reply:
You really want dot notation. As above, I'd go for `HasSum.exp`, and use `rexp` and `cexp` for the real and complex versions.

Reply:
Using `HasSum.exp` now (I'm already using `.rexp` for the real exponential version).

Reply:
Did you forget to push this change?

Reply:
Looks like I actually forgot to change it... Thanks for spotting it!